mobile/library/cc/stream_prototype.h

```diff
@@ -20,7 +20,7 @@ class StreamPrototype {
 public:
   StreamPrototype(EngineSharedPtr engine);
 
-  StreamSharedPtr start(bool explicit_flow_control = false);
+  StreamSharedPtr start(bool explicit_flow_control = false, uint64_t min_chunk_size = 0);
```

Comment:
nit: I wonder if we should use a different term than "chunk" since that has HTTP-specific meaning, which does not apply here. Maybe something like "min read size"?